src/lib/conventional-commits.ts

```diff
@@ -182,7 +183,17 @@ export default function createConventionalCommits() {
       output.appendLine(
         `commitMessage: ${JSON.stringify(commitMessage, null, 2)}`,
       );
-      const message = serialize(commitMessage);
+
+      let message = serialize(commitMessage);
+      const detectBreakingChange = configuration.get<boolean>(
+        'detectBreakingChange',
+      );
+      if (detectBreakingChange && /^BREAKING( |-)CHANGE: /gm.test(message)) {
+        output.appendLine(
+          'detectBreakingChange: Found `BREAKING CHANGE` in message.',
+        );
+        message = message.replace(': ', '!: ');
```

Comment:
😦It looks like a patch, not a designed feature.

It's better not to replace the string, but to update the serialization function.